roles/confluent.test/molecule/custom-user-plaintext-rhel/verify.yml

```diff
@@ -0,0 +1,277 @@
+---
+- name: Verify - Zookeeper User Change and custom logging path
+  hosts: zookeeper1-kafka-broker1
+  gather_facts: false
+  tasks:
+    - name: Set Zookeeper user to new user
```

Comment:
now you can customize the users one more time, keep the group the same and make sure the 770 still works